src/lambda/commands/deploySamApplication.ts

```diff
@@ -324,3 +330,36 @@ function getDefaultWindowFunctions(): WindowFunctions {
         showInformationMessage: vscode.window.showInformationMessage,
     }
 }
+
+function notifyDuringLongDeployment(deployParameters: DeploySamApplicationParameters): NodeJS.Timeout {
+    return setTimeout(async () => {
+        const cfnClient = new DefaultCloudFormationClient(deployParameters.region)
+        const stack = (await cfnClient.describeStacks(deployParameters.destinationStackName)).Stacks
```

Comment:
The cfn stack `deployParameters.destinationStackName` is not guaranteed to exist. Please always check for error conditions or at least have a way to surface failures in the logs. If this fails, the timer silently fails and we don't have visibility into it.

    stack trace: ValidationError: Stack with id feb2023h does not exist